Documentation/using-corert/optimizing-corert.md

```diff
@@ -37,3 +37,6 @@ No action is needed on Windows since the platform convention is to generate debu
 
 ## Advanced options 
 * `<IlcDisableUnhandledExceptionExperience>true</IlcDisableUnhandledExceptionExperience>`: disables code that prints stack traces for unhandled exceptions to the console.
+* `<IlcSystemModule>classlibmodule</IlcSystemModule>`: Name of the module which contains basic classes. When specified, disable automatic referencing of the `System.Private.CoreLib` and other libraries. 
```

Comment:
If you would like to write one, we can have a separate page for this. Something along the lines of "using CoreRT with alternative core libraries". On it's own, this option is not very helpful unless you know what you want to do with it. It's a _very_ advanced topic. Might make more sense to just implicitly document this in the zerosharp repo.

Reply:
I'm not qualified for that page yet. All I can do is follow your tutorial without real understanding how things working. I can add link for example of usage this particular setting which point to SeeSharpSnake or zerosharp that way it will be somehow documented, until somebody manage to write suggested page. If I take hat of `early adopter` the more I can get without looking at the code the better, and if I know about existence of some options, I can do research. If I do not know about options, it's much harder for me to start looking for things.

Reply:
Let's go with a link to ZeroSharp repo then. It's a pretty advanced topic that is likely going to be only interesting to 0.1% of existing .NET developers. It is what enables C# to do cool things though, so it's worth pointing it out somewhere.